allennlp/commands/bulk.py

```diff
@@ -2,19 +2,22 @@
 from contextlib import ExitStack
 import json
 import sys
-from typing import Sequence, Optional, IO
+from typing import Optional, IO
 
 from allennlp.service.servable import Servable, ServableCollection
 
+def add_bulk_subparser(parser: argparse._SubParsersAction) -> argparse.ArgumentParser:  # pylint: disable=protected-access
+    description = '''Run the specified model against a JSONL input file.'''
+    subparser = parser.add_parser(
+            'bulk', description=description, help='Run a model in bulk.')
+    subparser.add_argument('model', type=str, help='the name of the model to run')
```

Comment:
the input file contains actual inputs (the things you want to make predictions on), not parameters. I'll add another flag for parameter_file when I figure out how to use it